activerecord/lib/active_record/core.rb

```diff
@@ -279,12 +279,18 @@ def arel_table # :nodoc:
         @arel_table ||= Arel::Table.new(table_name, type_caster: type_caster)
       end
 
-      def arel_attribute(name, table = arel_table) # :nodoc:
+      # Returns an instance of <tt>Arel::Attributes::Attribute</tt> which represents the named column.
+      # For example:
+      #
+      #   Developer.where(Developer[:salary].not_eq(9000))
+      def [](name, table = arel_table)
         name = name.to_s
         name = attribute_aliases[name] || name
         table[name]
       end
 
+      alias :[] :arel_attribute # :nodoc:
```

Comment:
Should not the doc be reverse? I thought the point was to point people to use `Model[]` not `Model.arel_attribute`

Reply:
Yes.  I should not write code while in meetings 😬